modules/serverpod_auth/serverpod_auth_server/lib/src/business/user_images.dart

```diff
@@ -81,7 +81,7 @@ class UserImages {
     }
 
     // Store the image.
-    var path = 'serverpod/user_images/$userId-$version$pathExtension';
+    var path = p.join('serverpod','user_images','$userId-$version$pathExtension');
```

Comment:
These aren't paths in the file system, but are paths in S3 and cloud storage. They need to stay with the forward slash or it will break compatibility.